fastlane/Fastfile

```diff
@@ -31,8 +31,8 @@ end
 
 desc "Start a new release"
 lane :release do |options|
-  # Ensure we are on the main branch
-  ensure_git_branch(branch: 'main')
+  # Ensure we are on the develop branch
+  ensure_git_branch(branch: 'develop')
```

Comment:
Do we have a hotfix lane? 🤔 What if we want to make a release from a hotfix branch? Or this was not yet done in the script? Maybe we could also provide a "no_ensure_branch" like we did with the git_clean 🤔 

This is just a question, I'll approve it since this is out of scope of this PR 👍

Reply:
Nice one! I think we should maybe check if it is develop OR main